deeplearning4j-zoo/src/main/java/org/deeplearning4j/zoo/InstantiableModel.java

```diff
@@ -12,9 +12,7 @@
 
     public Model init();
 
-    public ModelMetaData metaData();
-
-    public ZooType zooType();
+    @Deprecated public ModelMetaData metaData();
```

Comment:
As a general rule: any time we add a ```@Deprecated``` tag we should have javadoc to explain why and/or what to use instead. Even better: add a date/version (pretty sure we're going with 1.0.0-beta for next release)

Reply:
See update.